### ISSUE TITLE
[A11yAudit] Update Search and Percentage Inputs text alternatives [II]

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3908]
**User Story**
As Spark we want to 

**Notes:**
Paciello Audit on Spark Design System
QLC-2 Icons - Search Input - percentage input
Images with missing or inadequate text alternatives

ISSUE DESCRIPTION
Text alternatives provide a text equivalent of visual information for users who can't see images. Without appropriate text alternatives, information in images is not accessible. With incorrect text alternatives, image-based information is confusing, misleading, and/or irrelevant.

RECOMMENDATIONS
Make sure that the alternative text for images is clear and unambiguous. The actual alternative text is dependent upon the context in which the image is being used, and what the image contains. The W3C HTML Specification offers guidance in this area.

For some of the images, such as the key icon in 22 - Icons, there will be various uses for the SVG component.

In cases where the image is purely decorative, use the aria-hidden="true" attribute so the image is ignored by screen readers, but is still visible on the screen. This is the case for the percentage icon that appears in samples 46-48, whose text alternative should instead be incorporated into the input label (e.g. "Interest Rate (%)").

Similarly, the search icon that appears in samples 55-57 is described via an aria-label on its enclosing <input> element and can therefore be marked as decorative using the aria-hidden="true" attribute.

The CSS pseudo content dollar icon that appears in samples 40-42 can remain essentially decorative without a text alternative but the currency symbol should be added to the input label (e.g. "Payment ($)".

If the image is meaningful and should be described, alternative text needs to be supplied. The SVG can be given a `role="img"` attribute as well as a focusable="false" attribute with a nested title element describing the image. Apply the `aria-hidden="true"` attribute to the internal `<use>` element.

**AC:**
This is for all Frameworks
- Make sure all icons on monetary input, percent input, and search input are correctly identified with Alt-text or hidden from screen readers as needed.
- All icons will be aria-hidden (maybe not for monetary because its a :before)

**Comment by ssabo1** [comment]
Confirmed: This has been added to Spark Design System Team Backlog.
